functionnal_tests/cypress/integration/account/account_main_page.js

```diff
@@ -8,9 +8,17 @@ describe('Account page', function () {
     cy.loginAs('users')
     cy.visit('/')
   })
-  it("should be able it access it through header's button", function () {
+  it("should be able it access it through header's button", () => {
     cy.get('[data-cy=menuprofil__dropdown__button]').click()
     cy.get('[data-cy=menuprofil__dropdown__account__link]').click()
     cy.get('.userinfo')
   })
+
+  it("Avatar should not be empty", () => {
```

Comment:
as from standard.js, strings must be single quoted
Also, the test seems to be about untrimmed strings, the test name doesn't reflect it